apps/server/package.json

```diff
@@ -16,6 +16,7 @@
     "eslint": "^7.32.0",
     "eslint-config-custom-server": "*",
     "tsconfig": "*",
-    "typescript": "^4.5.3"
+    "typescript": "^4.5.3",
+    "socket.io": "*"
```

Comment:
sweep: add socket.io to dependencies not devDependencies

Reply:
sweep:  leave the devDependencies as it is,  move only socket.io to dependencies